envergo/templates/haie/moulinette/result_base.html

```diff
@@ -19,7 +19,9 @@
   <ul class="fr-mb-3w">
     {% for field in moulinette.main_form %}
       {% block main_form_field %}
-        <li>{% field_summary field %}</li>
+        {% if not field.name in "department,element,travaux" %}
```

Comment:
Il me semble que la methode get_triage_params permet de lister les clefs du form de triage. Ce sera plus robuste qu'une liste en dure.
```suggestion
        {% if not field.name in moulinette.get_triage_params %}
```

Reply:
Ah mais oui, merci pour la suggestion !